src/Nethermind/Nethermind.Merge.Plugin/Data/ExecutionPayload.cs

```diff
@@ -70,12 +70,22 @@ public ExecutionPayload(Block block)
 
     public ulong Timestamp { get; set; }
 
+    private byte[][] _encodedTransactions = Array.Empty<byte[]>();
+
     /// <summary>
     /// Gets or sets an array of RLP-encoded transaction where each item is a byte list (data)
     /// representing <c>TransactionType || TransactionPayload</c> or <c>LegacyTransaction</c> as defined in
     /// <see href="https://eips.ethereum.org/EIPS/eip-2718">EIP-2718</see>.
     /// </summary>
-    public byte[][] Transactions { get; set; } = Array.Empty<byte[]>();
```

Comment:
There is direct usage of the setter in the code, so we need to reset _transactions